lib/landrush/action/common.rb

```diff
@@ -6,6 +6,7 @@ module Common
         'VagrantPlugins::ProviderLibvirt::Provider'    => :libvirt,
         'HashiCorp::VagrantVMwarefusion::Provider'     => :vmware_fusion,
         'VagrantPlugins::Parallels::Provider'          => :parallels,
+        'VagrantPlugins::HyperV::Provider'             => :hyperv,
```

Comment:
thanks :-)
I am going to cut a 1.1.1, which will work for virtualbox, libvirt, hyperv